src/js/redux/reducers/award/awardReducer.js

```diff
@@ -0,0 +1,20 @@
+/**
+ * awardReducer.js
+ * Created by Emily Gullo 01/23/2017
+ **/
+
+const initialState = [];
```

Comment:
Bad copy/pasta and understanding of the reducer setup.  Working on adjusting accordingly.